docs/src/api.md

```diff
@@ -8,10 +8,11 @@ In this page we list the remaining API functions, which constitute the bulk of A
 Besides the generic interface of [`AgentBasedModel`](@ref), and the [`StandardABM`](@ref), highlighted in the tutorial, there is also:
 
 ```@docs
-UnremovableABM
+EventQueueABM
```

Comment:
I think we should move `AgentBasedModel` here as well, and only expand the docstring of `StandardABM` in the tutorial. In the tutorial we just mention explicitly part of the interface by showing how it works in terms of examples. E.g., do something like `model[2]`. I can do this after this PR is merged.